eslint/babel-eslint-plugin/LICENSE

```diff
@@ -1,23 +0,0 @@
-Copyright (c) 2014-2015 Jason Quense <jason@quense.me>
```

Comment:
This license is different from the top-level one (different name), we should keeep it.